project-two/css/about-me.css

```diff
@@ -0,0 +1,310 @@
+/* DESKTOP NAV ADJUSTMENT */
+.nav-desktop-list-item-about-me {
+  color: var(--royal-blue);
+}
+
+/* MAIN */
+main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1rem 1rem 4rem 1rem;
+  background: linear-gradient(180deg, var(--white) 25%, var(--dark-beige) 25%);
+  /* horizontal split white / beige background */
+}
+
+@media screen and (min-width: 800px) {
+  main {
+    flex-direction: row;
+    justify-content: center;
+    align-items: flex-start;
+    gap: 3.5rem; 
+    /* gap between card one and card two */
+    padding: 12rem 0;
```

Comment:
Browsers have settings to zoom the whole page or to just increase the font size, if you set everything to rem you are zooming the page when the user only wanted to increase the font size.